### PR TITLE
Add supermanz-stark-bond.toml

### PR DESCRIPTION
**Pull request by supermanz56**
## Discord handler
< insert here you discord handler (if you have one) >

## Checklist before opening a pull request
- [x] My submissions follow the [instructions](https://hackmd.io/CNzB-rsNQ1ykRHkj8EDbTA?view)
- [x] Im aware my submission will not be modifiable/delatable if merged

### CODE REVIEW

**Review by Fraccaman — APPROVED** [review]

**Comment by Fraccaman** [comment]
there is an invalid character in the file submitted

**Comment by Fraccaman** [comment]
no its in the file ```unexpected character found: `\u{a0}` at line 510 column 67```

**Comment by supermanz56** [comment]
I'm stumped.

The file submitted has only 8 lines.  How can there be a problem at 'Line 510'?   I followed this tutorial to generate the file: https://www.youtube.com/watch?v=3CeiAcC8d-s

Nothing should be this difficult. Thanks for your efforts to assist - but I give up.

File: transactions/supermanz-stark-bond.toml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+[[bond]]
+source = "tpknam1qzjh7x7aax75ulnr39ny5eagu7jf6e62hhqh5wfg5d6slxwuugmwg5053yl"
+validator = "tnam1qydvhqdu2q2vrgvju2ngpt6yhrehu525pus6m28p"
+amount = "135.92"
+
+[bond.signatures]
+tpknam1qzjh7x7aax75ulnr39ny5eagu7jf6e62hhqh5wfg5d6slxwuugmwg5053yl = "signam1qrvrs6heq7smd3vrk2vvsyv5utpgwv6py75qxhxvs9wsgtrt49mepq46uplt5feszumx7z2qtasx9kknr5glh9cpq3x3wsja3jr5zcc2qeye2w"
```
Comment (Fraccaman) [inline transactions/supermanz-stark-bond.toml:7]: the problem is here. What you see in CI is line 510 when we try to merge this file into `transaction.toml`

**Comment by iskay** [comment]
Thanks for your interest! Since the deadline for pre-genesis was Oct 18 @ 1700 UTC, I'll close this PR. You can still delegate as soon as mainnet genesis launches using any staking interface.

**Pull request closed without merging**